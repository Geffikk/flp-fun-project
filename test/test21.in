S,X,A
a
S
S->AAA
S->X
A->AaA
X->a
