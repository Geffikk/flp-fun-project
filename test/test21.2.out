S,A,X,a',<aA>,<AA>
a
S
S->A<AA>
<AA>->AA
A->A<aA>
<aA>->a'A
a'->a
X->A<AA>
